core/src/test/java/org/elasticsearch/action/admin/cluster/node/tasks/TasksIT.java

```diff
@@ -145,6 +145,15 @@ public void testTaskCounts() {
         assertThat(response.getTasks().size(), greaterThanOrEqualTo(cluster().numDataNodes()));
     }
 
+    public void testTaskInfoToString() {
+        // Run only on data nodes
+        ListTasksResponse response = client().admin().cluster().prepareListTasks().setActions(ListTasksAction.NAME).get();
```

Comment:
can we rather have a unit test for this? we don't need to send out a request and read back the response right?